src/System.Data.SqlClient/src/System/Data/SqlClient/TdsParser.cs

```diff
@@ -1837,9 +1837,6 @@ internal bool TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataRead
                                             _statisticsIsInTransaction = true;
                                             break;
                                         case TdsEnums.ENV_COMMITTRAN:
-                                            // SQLHOT 483
-                                            //  Must clear the retain id if the server-side transaction ends by anything other
```

Comment:
That comment references some transaction ID code that isn't in Core, same for the comment below.
http://referencesource.microsoft.com/#System.Data/System/Data/SqlClient/TdsParser.cs,2084